operator-common/src/main/java/io/strimzi/operator/common/operator/resource/AbstractNonNamespacedResourceOperator.java

```diff
@@ -118,15 +119,40 @@ public AbstractNonNamespacedResourceOperator(Vertx vertx, C client, String resou
      * Deletes the resource with the given name
      * and completes the given future accordingly
      */
-    protected Future<ReconcileResult<T>> internalDelete(String name) {
+    private Future<ReconcileResult<T>> internalDelete(String name) {
+
```

Comment:
Why the empty lines here and at the end of the method while you have almost no empty lines within the code?